books-and-breakfast/app/MissionScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Dimensions, StyleSheet, Text, View } from 'react-native';
+import Carousel, { Pagination } from 'react-native-snap-carousel';
+
+interface Entry {
+  title: string;
+  subtitle?: string;
+  body: string;
+  link?: string;
+}
+
+const { width: screenWidth } = Dimensions.get('window');
+
+const entries: Entry[] = [
```

Comment:
it would prob be cleaner if you move this data to a separate file in the data folder